docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-apps-java/cloud-ysql-yb-jdbc.md

```diff
@@ -0,0 +1,111 @@
+- Java Development Kit (JDK) 1.8, or later, is installed. JDK installers for Linux and macOS can be downloaded from [Oracle](http://jdk.java.net/), [Adoptium (OpenJDK)](https://adoptium.net/), or [Azul Systems (OpenJDK)](https://www.azul.com/downloads/?package=jdk). Homebrew users on macOS can install using `brew install openjdk`.
+- [Apache Maven](https://maven.apache.org/index.html) 3.3 or later, is installed.
+
+## Clone the application from GitHub
+
+Clone the sample application to your computer:
+
+```sh
+git clone https://github.com/yugabyte/simple-java-app-yugabyte-cloud.git && cd simple-java-app-yugabyte-cloud
+```
+
+## Provide Yugabyte Cloud settings
+
+The application needs to establish a secure connection to your Yugabyte Cloud instance. To do this:
+
+1. Open the `app.properties` file located in the application `src/main/resources/` folder.
+
+2. Set the following configuration settings:
+
+    - **host** - the host name of your Yugabyte Cloud instance. To obtain the cluster host name, sign in to Yugabyte Cloud, select your cluster on the **Clusters** page, and click **Settings**. The host is displayed under **Network Access**.
+    - **port** - the port number that will be used by the JDBC driver (the default YugabyteDB YSQL port is 5433).
+    - **dbUser** and **dbPassword** - the username and password for the YugabyteDB database. If you are using the default database you created when deploying the cluster, these can be found in the credentials file you downloaded.
+    - **sslRootCert** - the full path to the cluster CA certificate
+
+3. Save the file.
+
+## Build and run the application
+
+To build the application, run the following command.
+
+```sh
+$ mvn clean package
+```
+
+To start the application, run the following command.
+
+```sh
+java -cp target/simple-java-app-yugabyte-cloud-1.0-SNAPSHOT.jar SampleApp
+```
+
+If you are running the application on a free or single node cluster, the driver will display a warning that the load balance failed and will fall back to a regular connection.
+
+You should then see output similar to the following:
+
+```output
+>>>> Successfully connected to Yugabyte Cloud.
+>>>> Successfully created DemoAccount table.
+>>>> Selecting accounts:
+name = Jessica, age = 28, country = USA, balance = 10000
+name = John, age = 28, country = Canada, balance = 9000
+
+>>>> Transferred 800 between accounts.
+>>>> Selecting accounts:
+name = Jessica, age = 28, country = USA, balance = 9200
+name = John, age = 28, country = Canada, balance = 9800
+```
+
+You have successfully executed a simple Java application that works with Yugabyte Cloud.
+
+## Explore the application logic
+
+Open the `SampleApp.java` file in the application `/src/main/java/` folder. The application has the following methods:
+
+- **main** establishes a connection with your cluster via the topology-aware Yugabyte JDBC driver.
```

Comment:
You know, let's add more visuals in the form of the code snippets here so that we don't require the developer to open the project with her/his IDE unless necessary. We don't need to do this in GitHub's readme but this doc page is a perfect place to be more descriptive.

For instance, for the **main** method, let's add this snippet:
`
YBClusterAwareDataSource ds = new YBClusterAwareDataSource();

        ds.setUrl("jdbc:yugabytedb://" + settings.getProperty("host") + ":"
            + settings.getProperty("port") + "/yugabyte");
        ds.setUser(settings.getProperty("dbUser"));
        ds.setPassword(settings.getProperty("dbPassword"));
        
        // Additional SSL-specific settings. See the source code for details.

        Connection conn = ds.getConnection();
`